src/main/java/github/weichware10/toolbox/gui/ConfirmBox.java

```diff
@@ -1,69 +1,39 @@
 package github.weichware10.toolbox.gui;
 
-import javafx.event.ActionEvent;
-import javafx.event.EventHandler;
-import javafx.geometry.Insets;
-import javafx.geometry.Pos;
+import java.io.IOException;
+
+import javafx.fxml.FXMLLoader;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'javafx.fxml.FXMLLoader'